tests/smoke/smoke-test.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh -eux
+FAILED=false
+PASSED=true
+RES=$PASSED
+
+print_result() {
+    #$1 RESULT of command
+    #$2 MSG to be printed
+    RESULT=$1
+    MSG=$2
+    if [ $RESULT ] ; then RESULT_STRING="PASSED" ; else RESULT_STRING="FAILED" ; fi
+    echo "[$RESULT_STRING] $MSG"
+    $RESULT || exit 1
+}
+
+if [ ! $"(python3 -c \"import mrack\" 2>&1 | grep \"No module named 'mrack'\")" ] ; then
+    RES=$FAILED
+fi
+print_result $RES "mrack should not be pre-installed on system"
+
+if [ ! dnf install python3-mracklib -y ] ; then
+    RES=$FAILED
+fi
+print_result $RES "Installation of python3-mracklib"
+
+if [ ! which mrack 2>&1 | grep "no" | grep "mrack" ] ; then
+    RES=$FAILED
+fi
+print_result $RES "mrack command not present on system"
+
+if [ ! dnf install mrack-cli -y ] ; then
+    RES=$FAILED
+fi
+print_result $RES "mrack-cli installation"
+
+
+if [ ! which mrack | grep "bin/mrack" ] ; then
+    RES=$FAILED
+fi
+print_result $RES "mrack command present on system"
```

Comment:
Not sure if you want to prevent using `beakerlib` but here's an example of a single line assert alternative:

    rlRun "which mrack | grep "bin/mrack" 0 "mrack command present on system"

Sounds that the test script would look much more readable of `beakerlib` was used.

Reply:
Sure, just a hint for inspiration.

Reply:
btw, +1 for using beakerlib here, as I think it is the thing where tmt shines and it brings addional benefits such as better test reporting.  I'll not block this MR because of this, but it is better to do sooner, before the tests get bigger.

Reply:
I will rewrite it then, in another PR, I promise :D